src/screens/ChannelListScreen/hooks/useFollowUser.ts

```diff
@@ -39,6 +39,12 @@ const useFollowUser = () => {
     return Boolean(followContext?.users?.find((user) => user?.user_id_followed === targetUser?.id));
   };
 
+  const isFollowingUser = (user_id_followed: string): boolean => {
+    return Boolean(
+      followContext?.users?.find((user) => user?.user_id_followed === user_id_followed)
+    );
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `isFollowingUser` seems to be redundant as it is doing the same thing as the `isFollowing` function. Both are checking if a user is being followed by searching for the user in the `followContext.users` array. Consider removing this redundancy to improve maintainability and readability.

```diff
-  const isFollowingUser = (user_id_followed: string): boolean => {
-    return Boolean(
-      followContext?.users?.find((user) => user?.user_id_followed === user_id_followed)
-    );
-  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->